tests/unit/models/generation/test_vllm_generation.py

```
@@ -1040,6 +1049,360 @@ def test_vllm_generate_text(cluster, tokenizer):
+        top_p=generation_config["top_p"],
+        # We want to test the actual train flow and how this is used. So we need to get logprobs here.
+        logprobs=True,
+        return_tokens_as_token_ids=True,
+        max_tokens=1,
+    )
+
+    _wait_for_vllm_http_server_spinup(base_urls[0])
+
+    # Check that the re-tokenized ids are the same with the reference and different without the reference.
+    # WITHOUT reference token IDs
+    response = requests.post(url=f"{base_urls[0]}/../tokenize", json=body)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)